include/deal.II/fe/fe_system.h

```diff
@@ -1068,11 +1066,6 @@ class FESystem : public FiniteElement<dim,spacedim>
     ~InternalData();
 
     /**
-     * Flag indicating whether second derivatives shall be computed.
-     */
-    const bool compute_hessians;
```

Comment:
This assumes that all finite element classes now compute Hessians analytically. Is this the case?

Otherwise, we could keep this code and only some elements use the space so provided.